src/core/create_routes.ts

```diff
@@ -3,7 +3,7 @@ import glob from 'glob';
 import { src } from '../config';
 import { Route } from '../interfaces';
 
-export default function create_routes({ files } = { files: glob.sync('**/*.+(html|js|mjs)', { cwd: src() }) }) {
+export default function create_routes({ files } = { files: glob.sync('**/*.*', { cwd: src() }) }) {
```

Comment:
I might be mistaken, but will not `'**/*.*'` include the folders as well? That might be desired?